test/git.yml

```diff
@@ -179,3 +179,30 @@
     - fail:
         msg: "File bar.html does not exist"
       when: st.stat.exists is not defined or st.stat.exists == False or st.stat.isdir == True
+
+- name: When deploying using Git (remote path for identity key)
+  hosts: all
+  vars:
+    ansistrano_deploy_via: "git"
+    ansistrano_git_repo: git@github.com:fjros/ansistrano-git-test.git
+    ansistrano_git_branch: master
+    ansistrano_git_repo_tree: foo
+    ansistrano_deploy_to: "/tmp/git/my-app.com"
+    ansistrano_git_identity_key_remote_path_local_test: "{{ ansible_env.PWD }}/deploy.key"
```

Comment:
Yep, you're right. It was just in case someone wants to run the tests locally, because I noticed that `.travis.yml` differentiated between `local` and `docker` environments.

I have no real preference on this, so please let me know whether I should remove the `local` case or leave it as is.